Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -474,6 +504,14 @@ public protocol CarPlayNavigationDelegate: AnyObject, UnimplementedLogging {
      - parameter canceled: True if the user dismissed the CarPlay navigation view controller by tapping the Cancel button; false if the navigation view controller dismissed by some other means.
      */
     func carPlayNavigationViewControllerDidDismiss(_ carPlayNavigationViewController: CarPlayNavigationViewController, byCanceling canceled: Bool)
+    
+    /**
+     Called when the CarPlay navigation view controller detects an arrival.
+     
+     - parameter carPlayNavigationViewController: The CarPlay navigation view controller that was dismissed.
+     - returns: A boolean value indicating whether to show an arrival UI.
+     */
+    func carPlayNavigationViewController(_ carPlayNavigationViewController: CarPlayNavigationViewController, shouldPresentArrivalUIFor waypoint: Waypoint) -> Bool
```

Comment:
We miss documentation for `waypoint` parameter here. Documentation for `carPlayNavigationViewController` doesn't seem to be accurate either.

Reply:
Thanks for the suggestion, I'll improve this part.